server/oauth2.go

```diff
@@ -245,6 +245,8 @@ type idTokenClaims struct {
 	Groups []string `json:"groups,omitempty"`
 
 	Name string `json:"name,omitempty"`
+
+	Login string `json:"dex.coreos.com/login"`
```

Comment:
@rithujohn191 what do you think of this claim name? We probably need to start namespacing our custom claims but I don't know what the correct pattern is.